packages/blade/src/components/Icons/_Svg/Rect/Rect.native.tsx

```diff
@@ -2,8 +2,8 @@ import type { ReactElement } from 'react';
 import { Rect as RectNative } from 'react-native-svg';
 import type { RectProps } from './types';
 
-const Rect = ({ height, width, rx, ry, x, y }: RectProps): ReactElement => {
-  return <RectNative height={height} width={width} rx={rx} ry={ry} x={x} y={y} />;
+const Rect = ({ height, width, fill, rx, ry, x, y }: RectProps): ReactElement => {
```

Comment:
So because of this, is there a chance that some icon colors were broken?

Reply:
Yup 😓